e2e/exhaustive_suite_test.go

```diff
@@ -85,52 +85,54 @@ func Exhaustive(t *testing.T) {
 			w := workload
 			d := deployer
 
-			t.Run(w.GetName(), func(t *testing.T) {
+			t.Run(w.GetName()+"-"+d.GetName(), func(t *testing.T) {
```

Comment:
Adding a function accepting a workload and deloyer and returning a test name would be little nicer.